Docs/source/usage/parameters.rst

```diff
@@ -183,9 +183,11 @@ Domain Boundary Conditions
 --------------------------
 
 * ``boundary.field_lo`` and ``boundary_field_hi`` (`2 strings` for 2D, `3 strings` for 3D)
```

Comment:
I am wondering with `warpx.do_pml` now from the implicit on to off value change, if the default value for these options should be pml?

Reply:
When we had discussed this offline, during the meeting, we had thought that the default should be PEC. (PR in progress for this capability )

Reply:
But, it would be good to revisit, because `do_pml` was always 1 until now

Reply:
Thank you @ax3l for the offline discussion. We are setting default field boundary to pml to ensure that the default behaviour remains the same before and after the PR.